charts/tf-controller/templates/rbac.yaml

```diff
@@ -185,6 +185,8 @@ subjects:
 - kind: ServiceAccount
   name: {{ include "tf-controller.serviceAccountName" . }}
   namespace: {{ .Release.Namespace }}
+{{- $allowedNamespaces := append .Values.runner.serviceAccount.allowedNamespaces .Release.Namespace -}}
+{{- range $allowedNamespaces }}
```

Comment:
Does this not create a conflict of duplicate objects or overwrite?

I think it would be better to move this `range` down under the `subjects` line.